packages/components/src/font-size-picker/README.md

```diff
@@ -10,57 +10,79 @@ The component renders a series of buttons that allow the user to select predefin
 import { FontSizePicker } from '@wordpress/components';
 import { withState } from '@wordpress/compose';
 
+...
 const MyFontSizePicker = withState( {
 	fontSize: 16,
-} )( ( { fontSize, setState } ) => { 
+} )( ( { fontSize, setState } ) => {
 	const fontSizes = [
-		{ shortName: 'S', size: 12 },
-		{ shortName: 'M', size: 16 }
+		{ name: 'Small', size: 12 },
+		{ name: 'Big', size: 26 },
 	];
 	const fallbackFontSize = 16;
-	
-	return ( 
-		<FontSizePicker 
-			fontSizes={ fontSizes } 
+
+	return (
+		<FontSizePicker
+			fontSizes={ fontSizes }
 			value={ fontSize }
 			fallbackFontSize={ fallbackFontSize }
-			onChange={ fontSize => setState( { fontSize } ) } 
+			onChange={ ( newFontSize ) => {
+				setState( { fontSize: newFontSize } );
+			} }
 		/>
-	); 
+	);
 } );
+
+...
+
+<MyFontSizePicker />
 ```
 
 ## Props
 
 The component accepts the following props:
 
-### fontSizes
+### disableCustomFontSizes
 
-An array of font size objects. The object should contain properties size, name, shortName.
-The property "size" contains a number with the font size value. The "shortName" property includes a small label used in the buttons. Property "name" is used as the label when shortName is not provided.
+If true it will not be possible to choose a custom fontSize, the user will be forced to pick one of the sizes passed in fontSizes.
```

Comment:
```suggestion
If `true`, it will not be possible to choose a custom fontSize. The user will be forced to pick one of the pre-defined sizes passed in fontSizes.
```